packages/shared/lib/appSettings.ts

```diff
@@ -9,7 +9,7 @@ import { AppSettings, AppTheme } from './typings/app'
 export const appSettings = persistent<AppSettings>('settings', {
     deepLinking: false,
     language: 'en',
-    theme: 'light',
+    theme: 'system',
```

Comment:
Question
Are there going to be different themes?
I think it simplifies the application if there was only one theme available, instead of having 2 and getting the one from the system

Reply:
yes!! task description suggests that we use `system`

Reply:
🤐